components/cron-pvc/cron-job/job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: cleanup-pvc-pipelinerun
+spec:
+  # Repeat each 5 min
+  schedule: "*/5 * * * *"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  suspend: false
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: pvc-cleaner
+          containers:
+          - name: cleanup-pvc-pipelinerun
+            image: docker.io/openshift/origin-cli
+            imagePullPolicy: IfNotPresent
+            command:
+            - /bin/sh
+            - -c
+            - |
+              echo "Clean up pvc subpaths..."
+
+              NAMESPACE=$(cat "/var/run/secrets/kubernetes.io/serviceaccount/namespace")
```

Comment:
nit: namespace is not needed at all in this script, it's automatically using current namespace.